instrumentation/graphql/README.md

```diff
@@ -41,6 +41,11 @@ OpenTelemetry::SDK.configure do |c|
     enable_platform_authorized: false,
     # enable_platform_resolve_type maps to the resolve_type and resolve_type_lazy keys
     enable_platform_resolve_type: false
+
+    # Controls if platform tracing (field/authorized/resolve_type)
+    # should use the legacy span names (e.g. "MyType.myField") or the
+    # new normalized span names (e.g. "graphql.execute_field").
+    legacy_platform_span_names: false
```

Comment:
I saw in the PR message that we would consider flipping this from `false` -> `true` if instrumentation ever hits 1.0 (we're not in a rush).

Why wait? It's not stable now; and if we think this is a useful enough change to flip at `1.0` why not now?

This is not a blocking comment, I'm just curious what you think here.

Reply:
Wasn't sure on this gem's policy on breaking changes. We can change the default to still give folks the option of using the old behaviour for now, no strong opinions.

Reply:
I don't think we have a super strong policy for things that are pre-1.0 honestly.